src/didymus/mc_reader.py

```diff
@@ -0,0 +1,63 @@
+# import here
+import numpy as np
+from didymus.pebble import Pebble
+
+
+# defining code-specific readers here:
+class OpenmcReader():
+    """
+    Class for reading Openmc pebble coordinates from the
+    OpenMC pack_spheres() function into and arrary of
+    distinct Pebble objects.
+    """
+
+    def __init__(self, coord_array):
+        '''
+        Initializes the OpenmcReader.
+
+        Parameters
+        ----------
+        coord_array : numpy.ndarray
+            A 2-D numpy array with shape (N, 3) of cartesian coordinates 
+            for the centroid of a sphere, obtained from the pack_spheres() 
+            function of OpenMC, where N is the number of spheres.
```

Comment:
```suggestion
            for the centroid of a sphere, obtained from the :function:`openmc.pack_spheres()`
            function, where N is the number of spheres.
```

Reply:
I am okay with applying this change, but I'll have to do it manually and change where the line breaks are, because the first one is 81 characters, which (I believe) breaks PEP8 (and I don't want the bot to come yell at me).